src/components/snapshots/useSnapshot.ts

```diff
@@ -68,9 +68,16 @@ const useSnapshot: SnapshotOperation = async ({
       // The quick way, using pg_restore (whole database only, no partial
       // exports)
       console.log('Restoring database...')
-      execSync(
-        `pg_restore -U postgres --clean --if-exists --dbname tmf_app_manager ${snapshotFolder}/database.dump`
-      )
+      try {
+        execSync(
+          `pg_restore -U postgres --clean --if-exists --dbname tmf_app_manager ${snapshotFolder}/database.dump`
+        )
+      } catch (err) {
+        console.log('There were errors dropping previous database objects. Trying again...')
+        execSync(
+          `pg_restore -U postgres --clean --if-exists --dbname tmf_app_manager ${snapshotFolder}/database.dump`
+        )
+      }
```

Comment:
Have added this for extra protection against #925 

Basically, I think most of the errors are actually fine, in that it just had errors dropping certain database objects. But to be sure, we re-run the restore command for any errors after the first attempt. If there are still errors, we can assume it's a legitimate "fail" and so throw error and stop restore process.

We need to test this a bit more. Please try loading a bunch of different snapshots.